people/dcallega: Add Davide’s personal page

dcallega opened the pull request:
Minimal content for my page

Commit:
people/dcallega: Add photos

Commit:
people/dcallega: Add personal page

Commit:
people/dcallega: Add biblio

igorburago commented on content/people/dcallega.md:
```diff
@@ -0,0 +1,96 @@
+---
+# Page front matter
```
Please remove the comments from the metadata in the front matter.

Commit:
people/dcallega: Removing comments from profile file

igorburago commented on content/people/dcallega.md:
```diff
@@ -1,93 +1,27 @@
-# the university, and year of graduation.
 education:
   - degree: M.S. in Computer Engineering
     institution: Università degli Studi di Padova
```
Let’s use English names of institutions, i.e., “University of Padova, Italy” in this case.

igorburago commented on content/people/dcallega.md:
```diff
@@ -1,93 +1,27 @@
     institution: Università degli Studi di Padova
     year: 2016
   - degree: B.S. in Information Engineering
     institution: Università degli Studi di Padova
```
Same here.

igorburago commented on content/people/dcallega.md:
```diff
@@ -1,93 +1,27 @@
-# interests. Keep each short and to the point; do not put a period
-# at the end.
 interests:
   - Distributed Machine Learning in Heterogeneous Computer Networks
```
Let’s not use title case in research interests; please don’t capitalize every word.

igorburago commented on content/people/dcallega.md:
```diff
@@ -1,93 +1,27 @@
 email: dcallega@uci.edu
-# Optional URL of your personal website (outside of the lab website).
-# Do not forget to include the protocol part, e.g., 'http://' or 'https://'.
 website: https://www.linkedin.com/in/davide-callegaro/
```
Website entry syntax [has changed][commit] to support (multiple) links with
titles. Please change this to:

    websites:
      - title: LinkedIn
        url: https://www.linkedin.com/in/davide-callegaro/

(In general, the `title` field is optional.)

[commit]: https://github.com/uci-iasl/website/commit/46182f701b9a2d85ced05103826c242ea412af3c

dcallega replied:
done

dcallega committed:
people/dcallega: Update on personal web page

De-capitalized research interests
Updated university name
Updated websites to new syntax (for multiple entries)

igorburago commented on content/people/dcallega.md:
```diff
@@ -6,19 +6,21 @@ yearjoined: 2016
-  - Reinforcement Learning and Semantic Maps
-  - Unmanned Aerial Vehicles
-  - Edge Computing
+  - distributed machine learning in heterogeneous computer networks
```
Keep the very first letter capitalized, of course.
E.g.: “Reinforcement learning and semantic maps”.

The point of my comment in the previous review was:
Just do not use title case; capitalize the words the same way you
would normally in a sentence.

igorburago commented on content/people/dcallega.md:
```diff
@@ -6,19 +6,21 @@ yearjoined: 2016
 education:
   - degree: M.S. in Computer Engineering
-    institution: Università degli Studi di Padova
+    institution: University of Padova
```
Please add the country: “University of Padova, Italy”.

dcallega committed:
people/dcallega: Update profile

igorburago reviewed (approved):
LGTM
